table/model.go

```diff
@@ -28,8 +29,21 @@ type Model struct {
 	missingDataIndicator interface{}
 
 	// Interaction
-	focused        bool
-	keyMap         KeyMap
+	focused bool
+	keyMap  KeyMap
+
+	// Taken from: 'Bubbles/List'
+	// Additional key mappings for the short and full help views. This allows
+	// you to add additional key mappings to the help menu without
+	// re-implementing the help component. Of course, you can also disable the
+	// list's help component and implement a new one if you need more
+	// flexibility.
+	// You have to supply a keybinding like this:
+	// key.NewBinding( key.WithKeys("shift+left"), key.WithHelp("shift+←", "scroll left"))
+	// It needs both 'WithKeys' and 'WithHelp'
+	AdditionalShortHelpKeys func() []key.Binding
```

Comment:
A full bare function here feels a little odd, but I do see how it's used in the list as well. Hmm.

One thing I would ask we update here if we go this route is to change this to unexported as `additionalShortHelpKeys`, then add `WithAdditionalShortHelpKeys(keys []key.Binding)` which internally then either just keeps a copy of the list or generates a function that returns a copy of the supplied list. I'd rather we keep the `With*` pattern to stay consistent in our own table.

This would also allow us to have a simpler interface with just supplying a list of extra bindings, rather than creating a function... is there a reason we need a function?

If we really want to add a function later I suppose we could then add `WithAdditionalShortHelpKeysFunc` as an advanced sort of feature, and still just keep the same underlying functionality with what's already implemented.

Reply:
Sorry I was a bit occupied last couple days. 

I do agree that we should change it to the 'With' style to keep it consitent. Will implement that and update the PR :).

Reply:
This should be fixed in the new Commit